src/Components/PaperEditor/TranscriptsContainer/SearchBar/index.js

```diff
@@ -1,9 +1,10 @@
 import PropTypes from 'prop-types';
```

Comment:
lots of refactoring and renaming, might be worth looking at the newest changes, instead of the older one.
looks like `showTextSearchPreferences` was redundant also.